File: DSA/Stuti_Jain_590012149/Day_9_Question_2.c
#include<stdio.h>
char s[100];
int i, v, c;
int main() {
    printf("Enter the string: \n");
    scanf("%s", &s);

    if(s[i]>='a' || s[i]<='z'){
    for(i=0; s[i]; i++){
        if(s[i]=='a' || s[i]=='e' || s[i]=='i' || s[i]=='o' || s[i]=='u'){
            v++;
    }
    else{
        c++;
    }

    }
    
    }

    printf("Vowels: %d\n", v);
    printf("Consonants: %d\n", c);

    return 0;

}

File: DSA/Sauhard_Semalty_590013630/Day9Question2.py
import sys

def vowel_and_consonant_count(string: str) -> tuple[int, int]:
    "Counts vowels and consonants in the string."

    string = string.lower()

    vowels = 0
    consonants = 0

    for c in string:
        if c.isalpha():  # Can also check ascii values directly using ord function
            if c in ('a', 'e', 'i', 'o', 'u'):
                vowels += 1
            else:
                consonants += 1

    return vowels, consonants


def main() -> None:
    "The entry point of the program."

    string = input("Enter string: ")
    if not string:
        print("Empty string is not allowed.")
        sys.exit(-1)

    vowels, consonants = vowel_and_consonant_count(string)
    print(f"Vowels: {vowels}, Consonants: {consonants}")


if __name__ == "__main__":
    main()
